README.md

```diff
@@ -0,0 +1,16 @@
+# Welcome to Quarkiverse!
```

Comment:
shouldnt this just link to the wiki
and add this doc content tothe home.md page?

Reply:
Ah right I'll fix this ;-)